PokemonUnity.Shared/Game/Game.cs

```diff
@@ -52,19 +54,19 @@ public partial class Game : UnityUtilityIntegration//: UnityEngine.MonoBehaviour
 	//	}
 	//}
 
-
 	/**/
 	//On Project start...
 	//all XML files are opened, and locked
 	//one by one, they're scanned and checked to see if they're up to date (match latest with compile time info)
 	//if not the latest, perform an update by fetching and downloading latest info...
 	//else continue, and load the XML data into the variables
+	static System.Data.SQLite.SQLiteConnection con  = new System.Data.SQLite.SQLiteConnection(@"Data Source=..\..\..\\veekun-pokedex.sqlite");
```

Comment:
sqlite isnt a part of the git repo, since project is already over 2gbs of storage data (which really needs to be cleaned out)... the sqlite db is another 50mbs additional, while veekun's csv is only 5 mbs, that users can recreate the sqlite with.